ajax/libs/Captionator/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Captionator",
+  "filename": "captionator-min.js",
+  "description": "Simple closed-captioning polyfill for HTML5. Just 8KB when gzipped!",
+  "version": "0.5",
+  "homepage": "http://captionatorjs.com/",
+  "author": {
+    "name": "Christopher Giffard",
+    "url": "https://github.com/cgiffard"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cgiffard/Captionator.git"
+  },
+  "keywords": [
+    "Captionator",
+    "captionator.js",
+    "js"
+  ],
+  "license": "BSD-2-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/cgiffard/Captionator.git",
+    "basePath": "js",
+    "files": [
+      "captionator*!(-example)"
```

Comment:
@PeterDaveHello I review it again. I don't know whether line 26 is okay. 
But @pvnr0082t said he use autoupdate then it worked.
Please check this pull request again, thank you.